docs/guides/authentication-and-authorisation.md

```diff
@@ -30,3 +31,14 @@ should be the same as the one you use to query the data. If they don't match
 you won't see any data. As of now, you can't see series from other tenants.
 
 For more information regarding the tenant ID limits, refer to: [Tenant ID limitations](./limitations.md#tenant-id-naming)
+
+### Cortex-Tenant
+
+One way to add `X-Scope-OrgID` to Prometheus requests is to use a [cortex-tenant](https://github.com/blind-oracle/cortex-tenant)
+proxy which is able to extract the tenant ID from Prometheus labels.
```

Comment:
For transparency, I would just explicitly mention that `cortex-tenant` is a community project, not maintained by Cortex maintainers.

Reply:
Yep, thought about this but wasn't sure how to say this properly. Done.